design/renaming.md

```diff
@@ -0,0 +1,59 @@
+# Renaming definitions
+
+These changes need to be applied across:
+- Package names and dependencies in `package.json` files
+- All help and error messages in CLI and lib
+- All files set up upon an init (including `.gitignore` contents)
+- All files in zepkit boxes
+- Source files
+- READMEs, HACKING, and other md files in the repo
+- All files in the documentation for the upcoming version
+
+We need to define how we want to support the migration from old to new config files. Depending on this, we may not need to rename all mock packages in our tests.
+
+## Package names
+
+| Current | New |
+|---|---|
+| zos | @openzeppelin/cli |
+| zos-lib | @openzeppelin/upgrades.js |
+| zos-docs | @openzeppelin/sdk-docs |
+
+## Internal folder names
+
+| Current | New |
+|---|---|
+| packages/cli | packages/cli |
+| packages/lib | packages/lib |
+
+## Commands
+
+| Current | New |
+|---|---|
+| **zos** init MyProject | **oz** init MyProject |
```

Comment:
- Support `zos` with a deprecation notice 
- Support both `oz` and `openzeppelin`

In all messages and docs, use `openzeppelin` instead of `oz`